### PR TITLE
Implement browser window existence monitoring

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @AlexanderMoskovkin @churkin

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -24,5 +25,6 @@ export default {
     isMaximized,
     screenshot,
     generateThumbnail,
-    isValidDeviceName
+    isValidDeviceName,
+    watchWindowExistence
```
Comment (VasilyStrelyaev) [inline src/index.js]: `watch...Existence` is not quite good. Can we go with just `watchWindow` or is it too vague?

Comment (AlexanderMoskovkin) [inline src/index.js]: `watchWindow` looks ok for me. What do you think @AndreyBelym @churkin

Comment (churkin) [inline src/index.js]: @AlexanderMoskovkin Fine

Comment (AlexanderMoskovkin) [inline src/index.js]: /cc @AndreyBelym

**Comment by AlexanderMoskovkin** [comment]
FPR /cc @VasilyStrelyaev 

@AndreyBelym please publish a new minor version when the PR's merged

**Pull request merged**